### PR TITLE
fix: Breaking install due to missing hook & setup server side CI (LAN-720)

### PR DESCRIPTION
**Pull request by marination**
- Hook `landa_custom_fields` is empty and so `create_custom_fields` gets an empty list instead of dict, which breaks install
- CI to ensure we're not merging breaking (literally) changes
- Custom API to complete setup wizard for landa test site before creating fixtures. Only invoked if site has testing enabled or is in a CI environment

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    name: Server
+
+    services:
+      mariadb:
+        image: mariadb:10.3
```
Comment (barredterra) [inline .github/workflows/ci.yml]:
```suggestion
        image: mariadb:10.5
```

We're running 10.5 in production

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12
```
Comment (barredterra) [inline .github/workflows/ci.yml]:
```suggestion
          node-version: 14
```

According to the docs, frappe v13 depends on node v14

Comment (marination) [reply .github/workflows/ci.yml]: big yikes my bad

**Comment by barredterra** [comment]
@marination looks like there's still a redis error while completing setup wizard. Should we ignore or fix this?

**Comment by marination** [comment]
I could not replicate this locally. Can you try `creating a test site > installing erpnext > enabling tests for site > installing landa` and see ? If that's too much I think we can ignore it. I'll take a look at it when we push tests for this app

**Review by barredterra — APPROVED** [review]

**Pull request merged**